Jenkinsfile

```diff
@@ -12,6 +12,11 @@ def getCommitInfo = {
   appGitCommitMessage = sh(returnStdout: true, script: "cd ${APP_DIRECTORY}; git log -1 --pretty=%B").trim()
 }
 
+def notifySlack(colour, buildStatus, infraGitCommitAuthor, gitCommit, gitCommitAuthor, gitCommitMessage, stageName) {
+  // call the global slackSend method in Jenkins
+  slackSend(color: colour, message: "${buildStatus}: ${infraGitCommitAuthor} [#${appGitCommit}] (<${appGitCommitAuthor} ${appGitCommitMessage}) ${stageName}", channel: '#psammead')
```

Comment:
It would be better to pass the channel as a parameter to this function, so that we can re-use this function in future / pull it out of Psammead into a shared library.

Reply:
Also, are there docs for the global methods available in Jenkins? Where can I inspect them?

Reply:
There are methods documented but can't put the links in a public repo :) I'll add to the infrastructure README